config/requirements.txt

```diff
@@ -0,0 +1,2 @@
+fypp
```

Comment:
Why is this file needed? What is its aim?

Reply:
Since we need here python for the preprocessing, I added the requirements file for installation of dependencies. Maybe I should add that to the README